### PR TITLE
feat(plugin): add mydata functionality

### PR DESCRIPTION
**Pull request by keiya01**

### CODE REVIEW

File: plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/WebDataTab.tsx [hunk 1]
```diff
@@ -30,8 +30,15 @@ const WebDataTab: React.FC<Props> = ({ onOpenDetails, setSelectedWebItem }) => {
...
       // more exceptions will be added in the future
       switch (extension) {
-        case "kmz":
-          return "kml";
+        // georss
+        case "rss":
+          return "rss";
+        // georss
+        case "xml":
+          return "xml";
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/WebDataTab.tsx:38]: is xml also georss?

Comment (keiya01) [reply plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/WebDataTab.tsx:38]: Yes!

File: plugin/web/extensions/sidebar/core/components/hooks.ts [hunk 2]
```diff
@@ -124,7 +124,7 @@ export default () => {
         let dataToAdd = data?.find(d => d.dataID === dataset.dataID);
 
         if (!dataToAdd) {
-          dataToAdd = convertToData(dataset);
+          dataToAdd = convertToData(dataset as DataCatalogItem);
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/hooks.ts:127]: Can you clean up this function (handleProjectDatasetAdd) based on the UserDataItem type update.
I misunderstood and thought myData data shouldn't be shown at all in the sidebar, and only added to the scene, so the first if statement does just that.

Comment (keiya01) [reply plugin/web/extensions/sidebar/core/components/hooks.ts]: I fixed!
[fix: type](https://github.com/eukarya-inc/reearth-plateauview/pull/146/commits/f9c30831402b82b529e3161ec9e844fa2eae35b9)

Comment (keiya01) [reply plugin/web/extensions/sidebar/core/components/hooks.ts:122]: I fixed.
[fix: remove if statement](https://github.com/eukarya-inc/reearth-plateauview/pull/146/commits/66986cb210016d07f15283039df1ad779ac55154)

**Review by KaWaite — APPROVED** [review]

**Pull request merged**